src/liquid/components/ld-tabs/ld-tabs.tsx

```diff
@@ -0,0 +1,97 @@
+import '../../components' // type definitions for type checks and intelliSense
+import { Component, Element, h, Host, Prop } from '@stencil/core'
+import { getClassNames } from '../../utils/getClassNames'
+
+let tabsCount = 0
+
+/**
+ * @virtualProp ref - reference to component
+ * @virtualProp {string | number} key - for tracking the node's identity when working with lists
+ */
+@Component({
+  tag: 'ld-tabs',
+  styleUrl: 'ld-tabs.css',
+  shadow: false,
+})
+export class LdTabs {
+  @Element() el: HTMLElement
+
+  /** Size of the tabs. */
+  @Prop() size?: 'sm' | 'lg'
+
+  /** Display mode. */
+  @Prop() mode?: 'ghost' | 'brand-color'
+
+  /** Sets border radii. */
+  @Prop() rounded?: 'all' | 'all-lg' | 'top' | 'top-lg'
+
+  private idDescriber = `ld-tabs-${tabsCount++}`
+
+  private updateTabs(currentLdTab) {
+    this.el
+      .querySelector('[aria-selected="true"]')
+      ?.closest('ld-tab')
+      .removeAttribute('selected')
+    currentLdTab.scrollIntoView({
```

Comment:
Is it possible to do the `scrollIntoView` inside the `ld-tab`'s `selected`-event-listener? Not sure, though, where's the right place to handle this.

Reply:
Since the tab needs to be scrolled into view not only on selection but also on focus you would have to additionally implement focus listeners on all tabs. I'd rather avoid that.